### ISSUE TITLE
POA calculation fails to project DNI into POA 🤦 

### ISSUE THREAD

**Issue by wholmgren** [SolarArbiter/solarforecastarbiter-core#718]
The Arbiter's power model fails to project the DNI into the plane of the array when computing POA irradiance:

https://github.com/SolarArbiter/solarforecastarbiter-core/blob/e02d458ba6635157fea0f62dc48dbf4bd67b5c75/solarforecastarbiter/pvmodel.py#L231-L233

With a quick skim of the function/module, you might see `aoi_modifier` and think POA is properly computed, but you'd be wrong. That modifier is simply the loss due to reflection off the glass. We're still missing the cosine projection. The line should read:

```python
aoi_loss = pvlib.iam.physical(aoi)
beam_effective = dni * np.cos(np.deg2rad(aoi)) * aoi_loss
poa_effective = beam_effective + poa_sky_diffuse + poa_ground_diffuse
```

Plot below shows the POA for a single axis tracker with and without the fix, plus POA calculated with ModelChain. I'm not sure about the remaining discrepancy with ModelChain as I thought I'd used consistent transposition and albedo values. So I still want to double check if there's another problem.

<img width="405" alt="Screen Shot 2021-08-21 at 1 24 55 PM" src="https://user-images.githubusercontent.com/4383303/130334020-56c285fe-280b-4032-bad9-a289bdfeb7b8.png">

Embarrassing oversight that's only now coming to light when experimenting with a clear sky power function and comparing to high quality power data. At least github added support for emojis in issue titles.

**Comment by wholmgren** [comment]
No additional calculation problem. I was comparing the Arbiter's `poa_effective` to pvlib's `ModelChain.total_irrad['poa_global']`. I should have compared to `ModelChain.effective_irradiance` (this is pvlib 0.8.0, so no `ModelChain.results.*`). Here's the proper comparison:

<img width="577" alt="Screen Shot 2021-08-21 at 6 33 32 PM" src="https://user-images.githubusercontent.com/4383303/130339043-406b2ccd-6f88-457b-bd46-ec4f1d5c2d52.png">
